### ISSUE TITLE
Shared noise for different cases

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#158915]
Testing #158759

`Error hints` and `No Inlay Hints on Line` have the same sound

**Comment by isidorn** [comment]
Good catch, but I think that is by design since VS Code stable behaves the same.
@hediet what is the exact use case when we play "no inlay hints on line"

**Comment by meganrogge** [comment]
Wouldn't that be confusing for someone who relies upon these sounds? I'd think they should be distinct

**Comment by hediet** [comment]
@jrieken added audio cues for inlay hints.

**Comment by jrieken** [comment]
We play the sound when users execute the "Read Line With Inline Hints" and we figured that the error sound is a good fit. @meganrogge do you see shared sound use as an issue or is this just an observation? To me it's the same as our repeated use of the info, warning, and error icons

**Comment by meganrogge** [comment]
IMO, having no inlay hints on a line should not be categorized as an error - since it's not one - just info.

**Comment by meganrogge** [comment]
in fact, does a screen reader / audio cue user even care about having no inlay hints on a line (aren't those just a visible feature)?

**Comment by jrieken** [comment]
> aren't those just a visible feature

Audio cues is a feature that can help vision impaired users, that's why there is an explicit "read line with inline hint" commands. Not doing anything for the "no hint" cause is bad because you don't know if the feature is broken or if there are nothing to read

**Comment by meganrogge** [comment]
I agree, but I am suggesting a unique noise - not removing it

**Comment by isidorn** [comment]
@jrieken the sound designer just created a "No Action Available" sound. Maybe this would be a better fit, and the sound that we can use for all the cases when there is no sound

Rename to .mp3
[No-Action-Avail_v5.txt](https://github.com/microsoft/vscode/files/9461644/No-Action-Avail_v5.txt)

fyi @AFre100

**Comment by AFre100** [comment]
Copy this -- good points all around -- will look into this :)
